calico_upgrade/pkg/migrate/migrate.go

```diff
@@ -125,42 +125,49 @@ func Validate(clientv3 clientv3.Interface, clientv1 clients.V1ClientInterface, i
 	status("Validating conversion of v1 data to v3")
 	data, err := queryAndConvertResources(clientv1)
 	if err != nil {
-		status("Error: unable to perform validation, please resolve errors and retry")
+		status("ERROR: unable to perform validation, please resolve errors and retry")
```

Comment:
not necessary, but wondering about passing in a `statusLevel` flag. If we decide to change the warning string again, we just do it in one place.